lib/KDropdownMenu/__tests__/KDropdownMenu.header.spec.js

```diff
@@ -0,0 +1,30 @@
+import { renderComponent, takeSnapshot, click } from '../../../jest.conf/visual.testUtils';
```

Comment:
Lets keep the test suite just as [the example I linked in the issue](https://github.com/learningequality/kolibri-design-system/blob/159e5a050f7c7a3b094c0802930e0ca1e4fc713e/lib/buttons-and-links/__tests__/KButton.spec.js#L69). So lets just have one KDropdownMenu.specs.js file, where we have this structure: 
```javascript
describe.visual('KDropdownMenu', () => {
  it(...visual test for test 1)
  it(...visual test for test 2)
  it(...visual test for test 3)
  it(...visual test for test 4)
})
```